### ISSUE TITLE
Midpoint Review - Accessibility Feedback - Benefits Management Tools Team 1, Claim Status Tool, Document Status Initiative

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#118761]
## Next Steps for the VFS team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/gb1dfnkl) at any time._

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket any time** after you've finished reviewing it. You don't need to confirm with Governance before closing.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should:** Use the new[ tabs component](https://design.va.gov/components/tabs) for the Status, Files, Overview subnavigation 
    - [ ] **OR Must:** Ensure that the subnavigation indicates the current page/screen programmatically (i.e. use `aria-current="page"`) to ensure assistive technology users receive an equivalent experience to non-assistive tech users.
- [ ] **Consider:** Ensure the placement of error type 2s are consistent, and generally follow design guidance on placement towards the top of the "section" (for non-global errors). This helps assistive technology users understand what is needed of them, or the status, in the current flow/task

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone
- [ ] Add the 'collaboration-cycle-mobile-app' label if this feedback is for the mobile app

**Comment by jstrothman** [comment]
Noting that I asked in the [midpoint design issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/118733)
> We've had tabs in place for quite some time. I'm concerned about the need to update our app to use the new tab component as part of this initiative, as opposed to implementing them as a separate initiative. As I understand it, the new tab components don't have discrete urls, and changing may result in some other adjustments we need to make in the app. Can you confirm if updating the tabs is a must for this initiative, or if it's something Governance would be comfortable with our crew committing to do within a specified timeframe?
